src/clients/SnapshotGraphql.ts

```diff
@@ -351,48 +350,50 @@ export class SnapshotGraphql extends API {
 
   async getVpDistribution(address: string, proposalId?: string): Promise<VpDistribution> {
     const query = `
-     query getVpDistribution($space: String!, $voter: String!, $proposal: String){
-        vp (
-          voter: $voter,
-          space: $space,
-          proposal: $proposal
-        ) {
-          vp
-          vp_by_strategy
-        } 
-      }
-    `
-    const variables = {
-      space: SNAPSHOT_SPACE,
-      voter: address,
-      proposal: proposalId || '',
-    }
+   query getVpDistribution($space: String!, $voter: String!, $proposal: String){
+     vp (voter: $voter, space: $space, proposal: $proposal) {
+       vp
+       vp_by_strategy
+     }
+   }
+ `
+
+    const variables = { space: SNAPSHOT_SPACE, voter: address, proposal: proposalId || '' }
 
     const result = await this.fetch<SnapshotVpResponse>(GRAPHQL_ENDPOINT, {
       method: 'POST',
-      json: {
-        query,
-        variables: variables,
-      },
+      json: { query, variables },
     })
 
-    if (!result?.data?.vp) {
-      throw Error('Unable to fetch VP Distribution')
-    }
+    if (!result?.data?.vp) throw Error('Unable to fetch VP Distribution')
+
+    const vpByStrategy = result.data.vp.vp_by_strategy
+    console.log('vpByStrategy', vpByStrategy)
+    const total = Math.floor(result.data.vp.vp)
 
-    const vpByStrategy = result?.data?.vp.vp_by_strategy
+    const I = {
```

Comment:
Maybe instead of having here we can update the StrategyOrder enum with this values.
This way we ensure that we use the correct order in all the app